courses/free-courses-en.md

```diff
@@ -235,8 +235,10 @@
 
 * [APL Course](https://course.dyalog.com) - Dyalog (:construction: *in process*)
 * [APL Cultivation](https://aplwiki.com/wiki/APL_Cultivation) - Adám Brudzewsky
+* [Aplwiki wiki](https://aplwiki.com/wiki/APL_Orchard) - aplwiki 
```

Comment:
APL Orchard: A Stack Exchange chat room for APL learning and teaching, not a free course.